cfgov/v1/forms.py

```diff
@@ -198,21 +198,19 @@ class FilterableListForm(forms.Form):
 
     def __init__(self, *args, **kwargs):
         parent = kwargs.pop('parent')
+        hostname = kwargs.pop('hostname')
         super(FilterableListForm, self).__init__(*args, **kwargs)
-        self.set_topics(parent)
-        self.set_authors(parent)
+        self.set_topics(parent, hostname)
+        self.set_authors(parent, hostname)
 
     # Populate Topics' choices
-    def set_topics(self, parent):
-        live_tags = [tag for tags in [page.tags.names() for page in
-                    AbstractFilterPage.objects.live().descendant_of(
-                    parent)] for tag in tags]
-        shared_tags = [tag for tags in [page.tags.names() for page in
-                       AbstractFilterPage.objects.descendant_of(parent)
-                       if page.shared] for tag in tags]
-        all_tags = list(chain(live_tags, shared_tags))
+    def set_topics(self, parent, hostname):
+        live_tags = [tag for tags in
```

Comment:
lets rename `live_tags` ==> `tags` since now they could be a combination of live/shared